Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Sunstrider CoA & Dynamic Quel'thalas Flag

## Changelog:
- Sunstrider dynasty has CoA.
- Quel'thalas (both High Kingdom and Kingdom) gets a red version of the flag when its ruler becomes of Sin'dorei culture.


# How to test:
- Play Quel'thalas: use `culture blood_elf` console command to convert to **Sin'dorei** culture, check the titles.
- Play any character, convert to **Sin'dorei** culture using `culture blood_elf` console command, usurp both Quel'thalas titles using `usurp e_quelthalas` and `usurp k_quelthalas`. Check the titles.
